README.md

```diff
@@ -159,6 +159,10 @@ If the request/response cycle completes before a response was sent to the
 client (for example, the TCP socket closed prematurely by a client aborting
 the request), then the status will be empty (displayed as `"-"` in the log).
 
+##### :colorstatus
```

Comment:
Currently they are just listed in alphabetical order, if you can move that up where "c" would go.